src/server/lib/test/TestIntrospectionRequest.js

```diff
@@ -0,0 +1,56 @@
+const Test = require('./Test.js');
```

Comment:
Questi file js in lib/test, ci servono per i test di Auth? In caso contrario rimuoviamoli dalla PR, li aggiungeremo eventualmente nei rilasci dedicati di Introspection e Revocation